accessibility-checker-engine/src/v2/dom/DOMMapper.ts

```diff
@@ -59,4 +64,28 @@ export class DOMMapper extends CommonMapper {
 
         return null;
     }
+
+    /**
+     * get real CSS bounds in css pixels, adjusted for scroll only
+     * @param node 
+     * @returns 
+     */
+    getBounds(node: Node) : Bounds {
```

Comment:
See note above regarding getBounds naming
```suggestion
    getUnadjustedBounds(node: Node) : Bounds {
```